back/tests/Integration/DatabaseTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Tests\Integration;
+
+abstract class DatabaseTestCase extends IntegrationTestCase
+{
+    protected function getPostByTitle($title) : array|false
+    {
+        $query = $this->pdo->prepare('SELECT * FROM POSTS WHERE title = :title');
+        $query->execute(['title' => $title]);
+        return $query->fetch();
+    }
+    protected function getPostById(int $id) : array|false
+    {
+        $query = $this->pdo->prepare('SELECT * FROM POSTS WHERE id = :id');
+        $query->execute(['id' => $id]);
+        return $query->fetch();
```

Comment:
It would be better to always use your Post object (or an array of Post objects) instead of returning arrays everywhere